lib/modules/dosomething/dosomething_api/includes/Transformer.php

```diff
@@ -83,39 +99,19 @@ protected function getPathParameters($parameters, $endpoint) {
 
 
   /**
-   * Calculate total number of Reportback items for the specified campaigns by
-   * each status requested.
-   *
-   * @param array $parameters Query parameters.
+   * @param $ids
```

Comment:
These functions should include documentation, esp around the datatypes of parameters, and their use.. https://www.drupal.org/coding-standards/docs#functions